### PR TITLE
[BTS-345] Apply theme props to <Tooltip />

### PR DESCRIPTION
**Pull request by allyssonsantos**

### CODE REVIEW

File: components/Tooltip/Tooltip.jsx [hunk 1]
```diff
@@ -86,11 +104,20 @@ Tooltip.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node,
   ]).isRequired,
+  theme: PropTypes.shape({
+    spacing: PropTypes.object,
+    colors: PropTypes.object,
```
Comment (ggdaltoso) [inline components/Tooltip/Tooltip.jsx]:
```suggestion
    colors: PropTypes.object,
    baseFontSize: PropTypes.number,
```

Comment (allyssonsantos) [reply components/Tooltip/Tooltip.jsx]: done

**Pull request merged**